### DISCUSSION TITLE
Add argumentFilled method to console commands

### DISCUSSION THREAD

**Discussion by farishrf** [laravel/framework#50710] [Ideas]
What do you think of `$this->argumentFilled('....')` in console commands? as a helper method
Use cases (just a random example):
```
public function handle(): void
{

if($this->argumentFilled('post')) {
 // do a particular thing if an argument is filled
}


// or

Post::query()
            ->when($this->argumentFilled('post')), function ($query, $postId) {
                return $query->where('id', $postId);
            })
// Iterate over all posts and do something.. ( or a specific post if argument filled)
}
```

**Comment by crynobone** [comment]
You should already be able to do the following:

```php
if (filled($this->argument('post')) {
 // do a particular thing if an argument is filled
}
```

**Reply by farishrf** [reply]
As you can see here from this commit:
https://github.com/laravel/framework/compare/11.x...farishrf:laravel-framework:feat/add-argument-filled-to-command
I already know `filled` exists, I just thought of this idea as a helper method which could make the code a bit readable.
